concordium-sdk/src/main/java/com/concordium/sdk/transactions/BlockItem.java

```diff
@@ -40,4 +57,28 @@ byte[] getBytes() {
         buffer.put(accountTransactionBytes);
         return buffer.array();
     }
+
+    public static BlockItem fromBytes(ByteBuffer source) {
```

Comment:
Any reason this is `public`? Currently only the `versionedXX` functions has been public to avoid confusing the integrator. 

Otherwise I think we should add to the documentation where the versioned/non versioned are expected from.

Reply:
I'm not a fan of hiding these things since they are perfectly respectable functions. But I have made this private now to be consistent with `getBytes`.